### PR TITLE
[40] Calculate Product Shipping Cost

### PR DESCRIPTION
**Pull request by faithakinbobola**
## Task: `Calculate Product Shipping Cost`
- `Added Shipping cost and Total cost to the product and made it responsive`

### CODE REVIEW

File: submissions/akinbobolaFaith/dom/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+var quantity = document.getElementById('quantity'),
+    price = document.getElementById('price'),
+    shipping = document.getElementById('shipping'),
+    cost = document.getElementById('cost'),
+    button = document.getElementById('myBtn')
+    unitPrice = 14000,
+    productPrice = unitPrice * quantity.value,
+    shipping = 0.15 * productPrice
+;
+console.dir(quantity);
```
Comment (ToluAkin) [inline submissions/akinbobolaFaith/dom/index.js:10]: Remove this output here @faithakinbobola

File: submissions/akinbobolaFaith/dom/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        quantity.nextElementSibling.innerHTML = "";
+    }
+}
+
+function priceField() {
+    if (quantity.value > 0) {
+        productPrice = unitPrice * quantity.value;
+        price.innerHTML = productPrice;        
+    }else {
+        return true;
```
Comment (ToluAkin) [inline submissions/akinbobolaFaith/dom/index.js:28]: this should return false if the quantity is less than 0 @faithakinbobola

File: submissions/akinbobolaFaith/dom/index.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+var quantity = document.getElementById('quantity'),
+    price = document.getElementById('price'),
+    shipping = document.getElementById('shipping'),
+    cost = document.getElementById('cost'),
+    button = document.getElementById('myBtn')
+    unitPrice = 14000,
```
Comment (ToluAkin) [inline submissions/akinbobolaFaith/dom/index.js:6]: You can't set unit prite here. You have to get it the same way you are getting the value of quantity @faithakinbobola

File: submissions/akinbobolaFaith/dom/index.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    }else {
+        return true;
+    }
+}
+
+function shippingAmount() {
+    if (productPrice < 1000) {
+        shipping = 100;
+        shippingCost.innerHTML = shipping;
+    } else if ((productPrice > 1000) && (productPrice < 20000)) {
```
Comment (ToluAkin) [inline submissions/akinbobolaFaith/dom/index.js:36]: `productPrice >= 1000 && productPrice <= 20000` @faithakinbobola same as below

File: submissions/akinbobolaFaith/dom/index.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+function totalCost() {
+    if (quantity.value > 0) {
+        cost.innerHTML = shipping + productPrice;
+    } else{
+        return true;
+    }
+}
+
+function indexHandle(e) {
+    quantityField(e);
```
Comment (ToluAkin) [inline submissions/akinbobolaFaith/dom/index.js:59]: You are not using event in any of the functions @faithakinbobola

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/akinbobolaFaith/dom/index.js [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    priceField(e);
+    shippingAmount(e);
+    totalCost(e);
+}
+
+quantity.addEventListener("change", indexHandle);
```
Comment (adaugochi) [inline submissions/akinbobolaFaith/dom/index.js:65]: Why are you listening for a `change` event?

File: submissions/akinbobolaFaith/dom/index.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
+let quantity = document.getElementById('quantity'),
+    price = document.getElementById('price'),
+    shipping = document.getElementById('shipping'),
+    cost = document.getElementById('cost'),
+    button = document.getElementById('myBtn'),
+    x = price.innerHTML,
```
Comment (ToluAkin) [inline submissions/akinbobolaFaith/dom/index.js:6]: I think you should rename this variable. It is not self - explanatory @faithakinbobola

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**